debian/docker.io.postinst

```diff
@@ -15,6 +15,10 @@ case "$1" in
 		# Ensure config directory permissions
 		#  On a fresh install, $2 = '' and dpkg "le-nl" treat that as a
 		#  greater version, so the body is not executed.
+                ### sergiodj: Maybe we can take this opportunity to
+                ### clean this up?  Version 1.11.2~ds1-1 doesn't seem
+                ### to be supported even in Bionic, which is almost
+                ### EOLing.
```

Comment:
Right, I think we can do it indeed. I did not pay too much attention to those cleanup opportunities to try to not increase the diff of what we have in the archive, but I think this one is really straightforward. 

+1 for this change.